en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md

```diff
@@ -78,6 +82,128 @@ Here are some of the graphs generated in Choreo cloud.
+
+```toml
+[analytics]
+ enabled = true
+ type = "elk"
+ 
+[analytics.enforcer]
+[analytics.enforcer.configProperties]
+ "publisher.reporter.class" = "org.wso2.am.analytics.publisher.sample.reporter.CustomReporter"
+```
+
+#### Step 1.2 - Enabling logs
+
+Update the `choreo-connect-1.x.x/docker-compose/choreo-connect(-with-apim)/conf/log4j2.properties` file as described below:
+<br/>
+
+There are two options to get the analytics logs as follows, 
+
+1. [Log to console]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics/#log-to-console)
+2. [Log to file]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics/#log-to-file)
+
+You need to configure the Filebeat agent based on the selected option.
+
+##### Log to console
+
+1. Add a reporter to the loggers list:
+<br/>
+```
+loggers = reporter, ... (list of other available loggers)
+```
+2. Add the following configurations after the loggers:
+<br/>
+```
+logger.reporter.name = org.wso2.am.analytics.publisher.reporter.elk
+logger.reporter.level = INFO
+logger.reporter.additivity = false
+logger.reporter.appenderRef.rolling.ref = ENFORCER_CONSOLE
+```
+
+##### Log to file
+
+1. Add `ENFORCER_ANALYTICS` to the appenders list:
+<br/>
+```
+appenders = ENFORCER_ANALYTICS, ... (list of other available appenders)
+```
+2. Add the following configuration after the appenders:
+</br>
+```
+appender.ENFORCER_ANALYTICS.type = RollingFile
+appender.ENFORCER_ANALYTICS.name = ENFORCER_ANALYTICS
+appender.ENFORCER_ANALYTICS.fileName = logs/enforcer_analytics.log
+appender.ENFORCER_ANALYTICS.filePattern = /logs/enforcer_analytics-%d{MM-dd-yyyy}.log
+appender.ENFORCER_ANALYTICS.layout.type = PatternLayout
+appender.ENFORCER_ANALYTICS.layout.pattern = [%d] - %m%ex%n
+appender.ENFORCER_ANALYTICS.policies.type = Policies
+appender.ENFORCER_ANALYTICS.policies.time.type = TimeBasedTriggeringPolicy
+appender.ENFORCER_ANALYTICS.policies.time.interval = 1
+appender.ENFORCER_ANALYTICS.policies.time.modulate = true
+appender.ENFORCER_ANALYTICS.policies.size.type = SizeBasedTriggeringPolicy
+appender.ENFORCER_ANALYTICS.policies.size.size=10MB
+appender.ENFORCER_ANALYTICS.strategy.type = DefaultRolloverStrategy
+appender.ENFORCER_ANALYTICS.strategy.max = 20
+appender.ENFORCER_ANALYTICS.filter.threshold.type = ThresholdFilter
+appender.ENFORCER_ANALYTICS.filter.threshold.level = DEBUG
+```
+3. Add a reporter to the loggers list:
+</br>
+```
+loggers = reporter, ... (list of other available loggers)
+```
+4. Add the following configurations after the loggers:
+<br/>
+```
+logger.reporter.name = org.wso2.am.analytics.publisher.reporter.elk
+logger.reporter.level = INFO
+logger.reporter.additivity = false
+logger.reporter.appenderRef.rolling.ref = ENFORCER_ANALYTICS
+```
```

Comment:
@suksw 
How come this section is different in 4.1.0 [1]? Which one is correct?

    ```
    logger.reporter.name = org.wso2.am.analytics.publisher.sample.reporter
    logger.reporter.level = INFO
    logger.reporter.additivity = false
    logger.reporter.appenderRef.rolling.ref = ENFORCER_ANALYTICS
    ```


[1] https://github.com/wso2/docs-apim/pull/5845/commits/196e35875a084000dcb5cacfc6209eebeaa6d048#diff-8d45b30154d44b477d372b5593c4a7cb408ba732b987753423c33722f236a07bR207-R211

Reply:
In the latest Choreo Connect we have changed the implementation. This change is not there in the CC 1.1.0(APIM 4.1.0). This content is correct.